### DISCUSSION TITLE
Please make user object optional

### DISCUSSION THREAD

**Discussion by makowskid** [cleaniquecoders/shrinkr#2] [Ideas]
Package is actually useless if I can't create generic short URLs, only user-specific.

**Comment by nasrulhazim** [comment]
> Package is actually useless if I can't create generic short URLs, only user-specific.

the intention of the package to have shortened url, own by user. it never meant for guest at the first place. This is, i believe similar to bit.ly. 

if you require null user's id, you always can update the migration when publish to nullable.
